src/monero_ux_nbgl_menu.c

```diff
@@ -83,32 +85,9 @@ static void update_account(void) {
     G_monero_vstate.disp_addr_m = 0;
 }
 
-static bool settings_navigation_cb(uint8_t page, nbgl_pageContent_t* content) {
-    if (page == 0) {
-        content->type = BARS_LIST;
-        content->barsList.barTexts = barTexts;
-        content->barsList.tokens = tokens;
-        content->barsList.nbBars = 3;
-        content->barsList.tuneId = TUNE_TAP_CASUAL;
-    } else if (page == 1) {
-        content->type = INFOS_LIST;
-        content->infosList.nbInfos = 3;
-        content->infosList.infoTypes = infoTypes;
-        content->infosList.infoContents = infoContents;
-    } else if (page == 2) {
-        content->type = INFOS_LIST;
-        content->infosList.nbInfos = 1;
-        content->infosList.infoTypes = infoTypes + 3;
-        content->infosList.infoContents = infoContents + 3;
-    } else {
-        return false;
-    }
-    return true;
-}
-
 static bool account_settings_navigation_cb(uint8_t page, nbgl_pageContent_t* content) {
     content->type = CHOICES_LIST;
-    content->choicesList.nbChoices = 5;
+    content->choicesList.nbChoices = 4;
```

Comment:
Why do we have 4 and no more 5? Is it linked to the split on 2 different pages?

Reply:
because it does not fit on Flex